.github/workflows/bump-auxiliary-packages.yml

```diff
@@ -0,0 +1,56 @@
+name: Bump Auxiliary Packages
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  update_generated_files:
+    name: Bump packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Github App Token
+        uses: mongodb-js/devtools-shared/actions/setup-bot-token@main
+        id: app-token
+        with:
+          app-id: ${{ vars.DEVTOOLS_BOT_APP_ID }}
+          private-key: ${{ secrets.DEVTOOLS_BOT_PRIVATE_KEY }}
+
+      - uses: actions/checkout@v4
+        with:
+          # don't checkout a detatched HEAD
+          ref: ${{ github.head_ref }}
```

Comment:
I think we may need to pass in `app-token.outputs.token` to the checkout action.

Reply:
Will add this but this is duplicated from https://github.com/mongodb-js/compass/blob/main/.github/workflows/bump-packages.yaml, do we need that there too?